.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing to Superwall
+
+We want to make contributing to this project as easy and transparent as
+possible, and actively welcome your pull requests. If you run into problems,
+please open an issue on GitHub.
+
+## Pull Requests
+
+1. Fork the repo and create your branch from `develop`.
+2. Run the setup script `./scripts/setup.sh`
+3. If you've added code that should be tested, add tests.
+4. If you've changed APIs, update the documentation.
+5. Ensure the test suite passes.
+6. Make sure your code lints (see below).
+7. Tag @yusuftor, @ianrumac @jakemor, and @anglinb in the pull request.
```

Comment:
You can change this to just you and me I think

Reply:
done 👍